_board/barduino.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: download
+board_id: "barduino"
+title: "Barduino Download"
+name: "Barduino"
+manufacturer: "Fablab Barcelona"
+board_url:
+ - "https://fablabbcn-projects.gitlab.io/electronics/barduino-docs/"
+board_image: "barduino.jpg"
+date_added: 2024-09-06
+family: esp32s3
+features:
+  - Breadboard-Friendly
+  - USB-C
+  - Bluetooth/BTLE
+  - Wi-Fi
+---
+
+Barduino is a development board created by Fablab Barcelona as a tool for it's educational programs.
```

Comment:
```suggestion
Barduino is a development board created by Fablab Barcelona as a tool for its educational programs.
```

Reply:
Nice catch, thank you. committed that fix.